miden/tests/integration/helpers.rs

```diff
@@ -140,6 +154,11 @@ impl Test {
         processor::execute(&script, &self.inputs)
     }
 
+    pub fn execute_iter(&self) -> VmStateIterator {
```

Comment:
Could we add a brief comment (something similar to the comments to similar functions here).

Reply:
fixed